test/common/router/scoped_rds_test.cc

```diff
@@ -420,7 +420,7 @@ name: foo_scoped_routes
         - match: {{ prefix: "/" }}
           route: {{ cluster: bluh }}
 )EOF";
-    if (!override_config_tmpl.empty()) {
+    if (fmt::basic_string_view<char>(override_config_tmpl).size() != 0) {
```

Comment:
IIUC according to the [docs](https://fmt.dev/latest/api.html#compatibility) this is for pre-C++17 compatibility.
Can the `route_config_tmpl` be changed to `string_view`, and the `override_config_tmpl` to an optional to string_view instead?